eclipse-scout-core/src/modeselector/Mode.less

```diff
@@ -31,45 +32,35 @@
 
   &:not(.last):not(.selected) {
     border-right: none;
-    padding-right: @mode-padding + 1px; /* account for the missing border, so that text does not jump */
+    padding-right: @mode-padding-x + @mode-border-width; /* account for the missing border, so that text does not jump */
   }
 
   &.after-selected {
     border-left: none;
-    padding-left: @mode-padding + 1px; /* account for the missing border, so that text does not jump */
+    padding-left: @mode-padding-x + @mode-border-width; /* account for the missing border, so that text does not jump */
   }
 
-  &:hover {
-    color: @mode-hover-text-color;
-  }
-
-  &:not(.disabled):not(.selected):active {
-    color: @button-active-color;
-    background-color: @button-active-background-color;
+  &.selected {
+    color: @default-button-color; // override button.selected
   }
 
-  &.selected {
-    color: @mode-selected-text-color;
-    background-color: @mode-selected-background-color;
-    border: none;
+  &.selected:hover {
+    color: @mode-selected-text-color; // override action:hover
   }
 
-  &:not(.disabled) {
-    cursor: pointer;
+  &:not(.disabled):not(.selected):hover {
+    color: @button-active-color; // override action:hover
   }
```

Comment:
currently it looks like the selected button can be clicked (hover and active states are applied). If clicking the selected button does not trigger an event, background-color should not be changed on hover/active